src/main/java/io/cdap/plugin/gcp/gcs/sink/DelegatingGCSRecordWriter.java

```diff
@@ -80,11 +80,10 @@ public void close(TaskAttemptContext context) throws IOException, InterruptedExc
       delegate.close(context);
     }
 
-    // Call the Commit Task and Commit Job implementations of this plugin to copy files into their final directory.
+    // Call the Commit Task implementation of this plugin to copy files into their final directory.
     // We need to do this at this stage because the OutputCommitter needs to be aware of the different partitions
     // that have been stored so far.
     delegatingGCSOutputCommitter.commitTask(context);
```

Comment:
It can go away, but reading from file and then creating outputCommitter have a lot of overheads, so will use the same committerMap to commit task and call it from here.

Reply:
removed and getting path from partitions.txt in commit task